boards/arm/ngultra_bring_up/doc/index.rst

```diff
@@ -0,0 +1,81 @@
+.. _ngultra_bring_up:
+
+NanoXplore NG-Ultra Bring-up
+############################
+
+Overview
+********
+
+The NG-Ultra bring-up is a demonstrator board showing NG-Ultra capabilities.
+The NG-Ultra is a radiation hardened by design SoC for aerospace applications.
+
+.. figure:: ngultra_bringup.png
+   :width: 600px
```

Comment:
please use jpeg and scale image so that there's no need to set a manual width. See https://github.com/zephyrproject-rtos/zephyr/pull/49024

Reply:
Done